src/main/java/com/programmerdan/minecraft/simpleadminhacks/hacks/GameTuning.java

```diff
@@ -325,16 +326,16 @@ public void onPlayerInteractEntity(PlayerInteractEntityEvent event) {
 	@EventHandler(priority = EventPriority.LOW, ignoreCancelled = true)
 	public void onEntityChangeBLock(EntityChangeBlockEvent event) {
 		if(config.isEnabled() &&
-				(!config.isEnderGrief() && event.getEntityType() == EntityType.ENDERMAN) ||
-				(!config.isWitherGrief() && event.getEntityType() == EntityType.WITHER)) {
+				(!config.isEnderGrief() && EntityType.ENDERMAN.equals(event.getEntityType())) ||
+				(!config.isWitherGrief() && EntityType.WITHER.equals(event.getEntityType()))) {
```

Comment:
What method does a dragon use? Any new mobs cause change events? Will look.

Reply:
no new mobs that I can see, but dragon easily added and withers can still destroy with wither skulls, so added that. Might need to gate wither skull damage and dragon fireball damage separately...

Reply:
Added dragon